assets/css/2-base/_navigation.sass

```diff
@@ -78,7 +78,7 @@
           opacity: 0
           transform: translateX(-3rem)
 
-          @for $i from 1 through 4
+          @for $i from 1 through 10
             &:nth-child(#{abs($i - 5)})
```

Comment:
Ah, this line will be a problem here actually,
I guess we have to loop from 1 to 5 and change this line to 
```javascript
            &:nth-child(#{abs($i - 6)})
```

Reply:
Alright, I don't know if it is me and my special relationship with sass, or the problem that I don't know how to build this jekyll site and test changes on my local machine beforehand.